hack/test-cmd.sh

```diff
@@ -74,6 +74,25 @@ function check-curl-proxy-code()
   echo "For address ${full_address}, got ${status} but wanted ${desired}"
   return 1
 }
+
+# TODO: Remove this function when we do the retry inside the kubectl commands. See #15333.
+function kubectl-with-retry()
+{
+  retry=true
+  count=0
+  ERROR_FILE="${KUBE_TEMP}/kubectl-error"
```

Comment:
It will be the same. But every run of test-cmd.sh shall get a different ${KUBE_TEMP}